tools/perf/profiles/brave-typical-android.zip.sizes

```diff
@@ -1,38 +1,38 @@
-Total size 60094K
+Total size 65749K
 
-GROUP Adblock [9053K total]
-   241K Brave Ad Block Updater (Brave First Party Adblock Filters (plaintext)) 1.0.407 [adcocjohghhfpidemphmcmlmhnfgikei]
-    58K Brave Ad Block Updater (Fanboy's Mobile Notifications (plaintext)) 1.0.14910 [bfpgedeaaibpoidldhjcknekahbikncb]
-  1479K Brave Ad Block Updater (EasyList Cookie (plaintext)) 1.0.16719 [cdbbhgbmjhfnhnmgeddbliobbofkgdhe]
+GROUP Adblock [9796K total]
+   241K Brave Ad Block Updater (Brave First Party Adblock Filters (plaintext)) 1.0.408 [adcocjohghhfpidemphmcmlmhnfgikei]
+    58K Brave Ad Block Updater (Fanboy's Mobile Notifications (plaintext)) 1.0.15037 [bfpgedeaaibpoidldhjcknekahbikncb]
+  1482K Brave Ad Block Updater (EasyList Cookie (plaintext)) 1.0.16875 [cdbbhgbmjhfnhnmgeddbliobbofkgdhe]
     97K Brave Ad Block Updater (Regional Catalog) 1.0.82 [gkboaolpopklhgplhaaiboijnklogmbc]
-  4378K Brave Ad Block Updater (Brave Default Adblock Filters (plaintext)) 1.0.14540 [iodkpdagapdfkphljnddpjlldadblomo]
-  1544K Brave Ad Block Updater (Brave Default Privacy Filters (plaintext)) 1.0.900 [kihnoaefogbkmblfimmibknnmkllbhlf]
-  1253K Brave Ad Block Updater (Resources) 1.0.111 [mfddibmblmbccpadfndgakiopmmhebop]
+  4390K Brave Ad Block Updater (Brave Default Adblock Filters (plaintext)) 1.0.14632 [iodkpdagapdfkphljnddpjlldadblomo]
+  1546K Brave Ad Block Updater (Brave Default Privacy Filters (plaintext)) 1.0.928 [kihnoaefogbkmblfimmibknnmkllbhlf]
+  1979K Brave Ad Block Updater (Resources) 1.0.112 [mfddibmblmbccpadfndgakiopmmhebop]
```

Comment:
@antonok-edm @ryanbr 
Do you know why `Brave Ad Block Updater (Resources)` size was increased from 1.2MB to 1.95MB?

Reply:
https://github.com/brave/brave-browser/issues/49051#issuecomment-3265254457